net.solarnetwork.node.weather.ibm.wc/build.properties

```diff
@@ -0,0 +1,4 @@
+source.. = src/
+output.. = build/eclipse/
+bin.includes = META-INF/,\
+               .
```

Comment:
This is missing a line for the `OSGI-INF/` path; in Eclipse there is a warning here, and if you click on the warning it will offer a fix for the issue that adds the missing line.